contracts/StorageManager.sol

```diff
@@ -114,38 +114,25 @@ contract StorageManager {
 
     /**
     >> FOR PROVIDER
-    @notice increases the capacity of a Offer.
-    @dev exercise caution with assigning additional capacity when capacity is already taken.
-    It may happen that when a lot of capacity is available and we release already-taken capacity, capacity overflows.
-    We explicitly allow this overflow to happen on the smart-contract level,
-    because the worst thing that can happen is that the provider now has less storage available than planned (in which case he can top it up himself).
-    However, take care of this in the client. REF_CAPACITY
-    @param increase the increase in capacity (in bytes).
+    @notice sets total capacity of Offer. Also add the difference from previous totalCapacity to availableCapacity
+    @param capacity the new capacity
     */
-    function increaseAvailableCapacity(uint128 increase) public existingOffer(msg.sender) {
+    function setTotalCapacity(uint128 capacity) public {
+        require(capacity != 0, "StorageManager: Capacity has to be greater then zero");
```

Comment:
Just to make sure I understand, this is just a "UX improving" check so that user on accident does not pause his own offer? So the only way to pause the offer now is to go through the `terminateOffer()` function, right?

Reply:
Yes :-)